style/theme.css

```diff
@@ -0,0 +1,187 @@
+        margin: 0 auto;
+    }
+    .block-region-bottom,
+    .block-region-footer {
+        clear: both;
+    }
+
+    #page-wrapper.region-block-elements #page.blocks-near-window {
+        padding-left: 15px;
+        padding-right: 15px;
+    }
+    .pagelayout-frontpage:not(.notloggedin) #page-wrapper.region-block-elements #page .main-content-region,
+    .pagelayout-mydashboard:not(.notloggedin) #page-wrapper.region-block-elements #page .main-content-region,
+    #page-course-view-topics:not(.notloggedin) #page-wrapper.region-block-elements #page .main-content-region {
+        text-align: center;
+    }
+    body.limitedwidth #page.drawers .main-inner {
+        text-align: left;
+    }
+    #page-wrapper.region-block-elements #page.blocks-near-window .main-region-block section.pre-side-block {
+        float: left;
+    }
+    #page-wrapper.region-block-elements #page.blocks-near-window .main-region-block section.post-side-block {
+        float: right;
+    }
+}
+
+@media (max-width: 1199px) {
+    .block-region-top,
+    .block-region-bottom,
+    .block-region-footer,
+    [data-region="blocks-column"] {
+        width: 100%;
+        padding: 0 20px;
+    }
+    .block-region-top {
+        padding: 0;
+    }
+    .block-region-headertop {
+        padding: 0 20px;
+    }
+    .main-region-block [data-region="blocks-column"] {
+        float: none;
+    }
+    #page-wrapper.region-block-elements #page {
+        padding: 0 15px;
+    }
+    #page.drawers .main-inner.main-content-region-block {
+        padding-left: 20px;
+        padding-right: 20px;
+    }
+    .block-region .block.card {
+        border-left: 1px solid rgba(0,0,0,.125);
+        border-right: 1px solid rgba(0,0,0,.125);
+    }
+}
+
+body #page-wrapper.region-block-elements #page .main-region-block .course-content {
+    text-align: left;
+}
+
+div#theme_boost-drawers-offcanvas {
+    z-index: 1050;
+}
+div#theme_boost-drawers-offcanvas .block-region-offcanvas {
+    z-index: 1053;
+}
+#page-wrapper.region-block-elements .navbar-nav .drawer-offcanvas-toggle {
+    display: flex;
+    align-items: center;
+}
+#page-wrapper.region-block-elements .navbar-nav .drawer-offcanvas-toggle .btn {
+    height: 40px;
+    background: none;
+}
+#page-wrapper.region-block-elements .navbar-nav .drawer-offcanvas-toggle .btn i {
+    font-size: 20px;
+    line-height: 30px;
+}
+
+/*off canvas*/
+div#theme_boost-drawers-offcanvas {
+    height: inherit;
+    min-height: 300px;
+    width: 100%;
+    z-index: 1050;
+}
+div#theme_boost-drawers-offcanvas .block-region-offcanvas {
+    z-index: 1053;
+}
+
+div.drawer.drawer-top {
+    visibility: hidden;
+    top: calc(-315px + -10px);
+
+}
+
+div.drawer.drawer-top.show {
+    visibility: visible;
+    top: 0;
+}
+.drawer-top .drawerheader .icon {
+    background: #000000;
+    padding: 2px;
+    color: #ffffff;
+}
+.drawer-top .drawerheader {
+    justify-content: right;
+    padding-right: 10px;
+}
```

Comment:
Please end the file with a single LF character see https://moodledev.io/general/development/policies/codingstyle#line-termination